Client/src/Remote/Endpoints/AuthenticationEndpoint.tsx

```diff
@@ -0,0 +1,20 @@
+import { ISignInUserModel, ISignUpUserModel } from "../../Models/Authentication/ISignUpModel"
+import { IUserAccountModel, IUserAccountRemoteModel, toLocalUserAccountModel } from "../Models/IUserAccountModel"
+
+import { post } from "../RemoteHelper"
+
+const signupRoute = '/signup'
+const loginRoute = '/login'
+const resetPasswordRoute = '/resetPassword'
+const updateUserInfoRoute = '/updateUserInfo'
+const userDetailsRoute = '/userDetails'
+
+export const callSignUp = async (signUpInfo: ISignUpUserModel): Promise<any> => {
+  return post(signUpInfo, signupRoute)
+}
+
+export const callLogIn = async (signInUser: ISignInUserModel): Promise<IUserAccountModel> => {
+  //server sets a token in browser cookie in http only mode
+  const remoteUser: IUserAccountRemoteModel = await post(signInUser, loginRoute)
```

Comment:
Do any subsequent requests to protected routes work after this?
For example, file upload?

Reply:
Yes, modified the backend to read the token from a cookie, tested, backend is able to evaluate the token in VerifyJWT